janus_core/single_point.py

```diff
@@ -20,22 +29,22 @@ class SinglePoint:
     architecture : Literal[architectures]
         MLIP architecture to use for single point calculations.
         Default is "mace_mp".
-    device : Literal[devices]
+    device : Devices
         Device to run model on. Default is "cpu".
-    read_kwargs : Optional[dict[str, Any]]
+    read_kwargs : ASEReadArgs
         Keyword arguments to pass to ase.io.read. Default is {}.
-    calc_kwargs : Optional[dict[str, Any]]
+    calc_kwargs : ASEReadArgs
```

Comment:
```suggestion
    calc_kwargs : Optional[dict[str, Any]]
```
Unless we want CalcOptArgs